pages/ae/apim/apim-installation.adoc

```diff
@@ -83,5 +83,45 @@ alerts:
       bulkEventsWait: 100    # Set the duration to wait for bulk events to be ready for sending. When set to 100ms with event size of 100, it means that we will wait for 100 events to be ready to be sent during 100ms. After this period of time, events will be sent event if there are less than 100 events to send.
 ```
 
-CAUTION: By default, to keep the same behavior of the previous version, events are sent over a websocket connection.
-The default behavior will switch to http in a next future version.
+[label label-version]#New in version 3.20.0#
```

Comment:
```suggestion
[label label-version]#New in version 2.0.0#
```
It's related to the connector version isn't it ?

Reply:
This is related to the APIM version